bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java

```diff
@@ -518,6 +519,48 @@ public void mutateRow(RowMutations rowMutations) throws IOException {
     }
   }
 
+  // Invoked via ByteBuddy
+  // Implements Result mutateRow(RowMutations) for HBase >= 2.4
+  public Result mutateRowResult(RowMutations rowMutations) throws IOException {
+    LOG.trace("mutateRow(RowMutation)");
+    if (rowMutations.getMutations().isEmpty()) {
+      return null;
```

Comment:
should we throw a invalid argument exception?

Reply:
yep empty mutation is ok with empty mutations